### ISSUE TITLE
Multiple inheritance does not work

### ISSUE THREAD

**Issue by ChristineTChen** [pawamoy/mkdocstrings#113]
**Please open an issue on [pytkdocs](https://github.com/pawamoy/pytkdocs/issues) instead
if this is related to Python docstrings parsing or the collection of Python objects!**

**Describe the bug**
If a python class inherits from multiple classes (parent `A` and parent `B`), the functions and corresponding docstrings from either parent class `A` and `B` do not render.

```
::: datadog_checks.base.checks.openmetrics.OpenMetricsBaseCheck
    rendering:
      heading_level: 4
    selection:
      members:
        - __init__
        - get_scraper_config <-- from OpenMetricsBaseCheck
        - get_http_handler <-- from parent class A
        - gauge <-- from parent class B
```

Only `__init__` and `get_scraper_config` will render.

**To Reproduce**
1. With a class that inherits from more than 1 class, select a function from one of the parent classes to render.
2. Build documentation and serve
3. See function and docstring from parent class is not rendered

**Expected behavior**
Expect to see functions from all parent classes rendered

**Screenshots**

**Information (please complete the following information):**
- OS: macOS
- Browser: chrome
- `mkdocstrings` version: 0.11.0

**Additional context**

**Comment by pawamoy** [comment]
Inheritance doesn't work at all :wink:

This is tracked in #78, therefore I'll close this as duplicate!

**Comment by pawamoy** [comment]
Hey @ChristineTChen, I just released mkdocstrings v0.11.4, which accepts pytkdocs v0.5.0, which adds support for class inheritance / inherited members. Could you try again to generate documentation for multi-inheriting classes with those versions and report any problem arising :slightly_smiling_face:? Thanks!
